docs/webactions.md

```diff
@@ -102,11 +102,13 @@ function main(params) {
     return {
         statusCode: 200,
         headers: { 'Content-Type': 'application/json' },
-        body: new Buffer(JSON.stringify(params)).toString('base64'),
+        body: params
     };
 }
 ```
 
+The default content-type for an HTTP response is `application/json` and the body may be any allowed JSON value. The default content-type may be omitted from the headers.
```

Comment:
Content is still sent in base64 format for raw http actions. This is not affected by this pr.